src/services/wallet.service.js

```diff
@@ -64,6 +64,9 @@ export async function setupWalletDir() {
 }
 
 export async function getMnemonic(){
+    //right now we are hardcoding 24 words
+    //  lets pass in a number. maybe we use a set of word lengths
+    //  12, 15, 18, 21, 24
     const { stdout, stderr } = await cmd(getMnemonicCmd(24));
```

Comment:
@lisandroiraguen  Alright so i was able to generate a wallet with a 9 word mnemonic. the trick is letting the mnemonic generator know you want to get a 9 word mnemonic. The requirement is really only that the word length is divisible by 3

Reply:
I see why it doesn't work with 9 words, lets do some changes then >D